doc/ref_model/chapters/chapter05.md

```diff
@@ -88,8 +88,8 @@ The following sections detail the NFVI SW profile features per type of virtual r
 
 | .conf | Feature | Type  | Description |
 |------------------|----------------|----------------|------------------------------------------------------------------------------------------------|
-| nfvi.net.cfg.001 | vNIC interface | IO virtualisation | e.g. virtio1.1, i40evf (Intel driver for VF SR-IOV). |
-| nfvi.net.cfg.002 | Overlay protocol | Protocols | The overlay network encapsulation protocol needs to enable ECMP in the underlay to take advantage of the scale-out features of the network fabric. |
+| nfvi.net.cfg.001 | vNIC interface | IO virtualisation | e.g. virtio1.1, i40evf (Intel driver for VF SR-IOV). <br> **Note:** This feature is not applicable to containers or Kubernetes based infrastructures. |
+| nfvi.net.cfg.002 | Overlay protocol | Protocols | The overlay network encapsulation protocol needs to enable ECMP in the underlay to take advantage of the scale-out features of the network fabric. <br> **Note:** In Kubernetes based infrastructures network separation is possible withtout an overlay (e.g.: with IPVLAN) |
```

Comment:
@CsatariGergely  Also, as I'm sure you know, this is a hot topic and may change, but keeping that debate separate, prefer to try to keep the table clean by using a footnote, as you've used in other places. Thx